### ISSUE TITLE
Add styling to the cgpa-calculator

### ISSUE THREAD

**Issue by niranjank2022** [niranjank2022/Cluster-of-Projects#1]
File to be taken care of: Cluster-of-Projects/Simple/cgpa-calculator/
Try to come up with unique designs.
To view the current code, you can use this link https://niranjank2022.github.io/Cluster-of-Projects/Simple/cgpa-calculator/
If you want to check your code changes, you can download Live Server VS Code Extension. ritwickdey.LiveServer

**Comment by musaddiq-ashfaq** [comment]
assign it to me

**Comment by niranjank2022** [comment]
Yes you can, work on it and submit the PR

**Comment by PrathameshhW** [comment]
Can I work on it? Let's compare the work of @musaddiq-ashfaq and merge the better styling.

**Comment by niranjank2022** [comment]
Yeah, both can work on the styling. Since this is a feature, we can have multiple themes, leaving the users with choices. And, if you want you can also try to come up with design ideas for the [Calculator](https://github.com/niranjank2022/Cluster-of-Projects/tree/main/Simple/Calculator) :)

**Comment by Sky-De** [comment]
@niranjank2022 if it is not done yet i would like to design that for you, if you are interested assigned this to me 
(As soon as i get assigned for this i will start)
Thanks

**Comment by niranjank2022** [comment]
> @niranjank2022 if it is not done yet i would like to design that for you, if you are interested assigned this to me 
> (As soon as i get assigned for this i will start)
> Thanks

Hello @Sky-De! Appreciate your interest for working on this. You can proceed and submit the PR. We can have multiple layouts for the user to choose.
